static/js/services/enketo.js

```diff
@@ -21,6 +22,7 @@ angular.module('inboxServices').service('Enketo',
     var objUrls = [];
     var xmlCache = {};
     var FORM_ATTACHMENT_NAME = 'xml';
+    var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
```

Comment:
Right. I wanted to export it from the Enketo service but also access it locally. `this` changes all the time so couldn't reliably be used, so I had to settle on this double assignment.